app/views/groups/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="navbar-cont flex">
+  <% if user_signed_in? %>
+    <%= link_to image_tag("https://i.postimg.cc/R0nhQZpQ/transparent-disconnect-icon-logout-sign-icon-interface-icon-5fcbe43d729733-9456472816071977574694.png' border='0' alt='transparent-disconnect-icon-logout-sign-icon-interface-icon-5fcbe43d729733-9456472816071977574694"), destroy_user_session_path, method: :delete, class:'back-arrow' %>
+  <% else %>
+    <%= button_to "Log in", new_user_session_path, class:"btn navbar-btn" %>
+    <%= button_to 'Sign up', new_user_registration_path, method: :get,class:"btn navbar-btn" %>
+  <% end %>
+  <h1>Categories</h1>
+  <img src="https://i.postimg.cc/R0xX4NtN/pngfind-com-lupa-png-sin-fondo-4235556.png' border='0' alt='pngfind-com-lupa-png-sin-fondo-4235556'" class="nav-img" />
+</nav>
+
+
+<h2 class="categories-title">Total payments: $<%= total_sum = Payment.sum(:amount) %></h2>
+
+<% if @groups.blank? %>
+  <h2 class="no-title"> No categories yet </h2>
+  <h4 class="no-title"> Your new categories will be placed here </h4>
+<% else %>
+  <div class="cards-container flex">
+    <% @groups.each do |category| %>
+      <div class="card flex">
+        <div class="card-header flex">
+          <div class="card-icon">
+            <%= image_tag category.icon rescue image_tag('https://img.icons8.com/?size=512&id=kLORTzuNOM2d&format=png', alt: "Category Icon") %>
+          </div>
+          <div class="card-info">
+            <h2><%= link_to category.name, group_payments_path(group_id: category.id) %></h2>
+            <p><%= l category.created_at.to_date, format: :long %></p>
+          </div>
+        </div>
+        <div class="card-footer flex">
+          <p>$<%= Payment.where(group_id: category.id).sum(:amount) %></p>
+          <%= button_to 'Delete', group_path(id: category.id), method: :delete%>
+        </div>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+
+<%= link_to "Add a Category", new_group_path, class:'link-to flex' %>
```

Comment:
- [ ] It would be nice if you could style this button to position it at the bottom of the page just as it is in the original design. 👍🏽


**Your Design**

<img width="457" alt="Screenshot 2023-08-16 at 2 49 55 PM" src="https://github.com/hernandanielzamora/How-Much/assets/50941074/087ce370-5526-4553-ab46-a00f363c5ad8">

**The Original Design**

<img width="342" alt="Screenshot 2023-08-16 at 2 43 01 PM" src="https://github.com/hernandanielzamora/How-Much/assets/50941074/4e624c4c-eee2-419c-a11d-fadf877634a2">